plugins/memcached/lib_memcached.ecllib

```diff
@@ -37,4 +37,6 @@ export memcached := SERVICE : plugin('memcached'), namespace('MemCachedPlugin')
   BOOLEAN Exists(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MExists';
   CONST VARSTRING KeyType(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partitionKey = '') : cpp,action,context,entrypoint='MKeyType'; //NOTE: calls get
   Clear(CONST VARSTRING options) : cpp,action,context,entrypoint='MClear';
+  Delete(CONST VARSTRING key, CONST VARSTRING options, CONST VARSTRING partionKey = '') : cpp,action,context,entrypoint='MDelete';
```

Comment:
The more I think about it, the less I like it (so if I said it was the right thing to do before, I was wrong). Providing undocumented functions is a bad precedent to set - and if we simultaneously try to match function names to the internal commands of every system we target AND with all other plugins, we will end up with 10 different names for every function in every plugin.

I think the function should be called delete, and should be documented. I also think the redis one should be called delete, and should be documented, though I don't really care if it's called del. But we should decide one way or the other - are we matching the internal names (generally we don't) or the standard ECL usage (generally we do and should)? We can't/shouldn;t try to do both.